spec/support/matchers/wait_for_ajax.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+puts "Loading spec/support/matchers/wait_for_ajax.rb"
+
+module WaitForAjax
```

Comment:
I think you need to call `include WaitForAjax` in the specs where you want to use this.
Also, I'd put this under `spec/support` directory, since this is not a matcher